tool/tctl/common/resource_command.go

```diff
@@ -586,6 +588,38 @@ func (rc *ResourceCommand) createBot(ctx context.Context, client auth.ClientI, r
 	return nil
 }
 
+func (rc *ResourceCommand) createDatabaseObjectImportRule(ctx context.Context, client auth.ClientI, raw services.UnknownResource) error {
+	rule, err := services.UnmarshalDatabaseObjectImportRule(raw.Raw)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	ruleName := rule.GetMetadata().GetName()
+
+	existingRule, err := client.DatabaseObjectImportRuleClient().GetDatabaseObjectImportRule(ctx, &dbobjectimportrulev1.GetDatabaseObjectImportRuleRequest{Name: ruleName})
+	if err != nil && !trace.IsNotFound(err) {
+		return trace.Wrap(err)
+	}
+
+	if existingRule != nil {
+		if !rc.force {
+			return trace.AlreadyExists("rule %q already exists", ruleName)
```

Comment:
If `CreateDatabaseObjectImportRule` already returns an error if rule already exists, why don't we use it? Otherwise, we have a race condition here and there is a case where we don't create an object even if `force` is specified (If a Get operation returns nothing, then something appears in the backend, and only then we call the Create RPC.)